packages/compiler/test/checker/relation.test.ts

```diff
@@ -446,6 +450,29 @@ describe("compiler: checker: type relations", () => {
     });
   });
 
+  describe("decimal target", () => {
+    it("can assign decimal", async () => {
+      await expectTypeAssignable({ source: "decimal", target: "decimal" });
+    });
+    it("can assign decimal128", async () => {
+      await expectTypeAssignable({ source: "decimal128", target: "decimal" });
+    });
+    it("can assign numeric literals", async () => {
+      await expectTypeAssignable({ source: "-2147483448", target: "decimal" });
```

Comment:
Should we add digits beyond the decimal point? (e.g. -5.50, 10.25)

Reply:
Good call, I'll add one